### PR TITLE
Fix available modules list not updated after add and delete command

### PR DESCRIPTION
**Pull request by Hilda-Ang**

### CODE REVIEW

File: src/main/java/seedu/planner/model/ModelManager.java [hunk 1]
```diff
@@ -40,6 +43,8 @@ public ModelManager(ReadOnlyModulePlanner modulePlanner, UserPrefs userPrefs) {
         logger.fine("Initializing with planner: " + modulePlanner + " and user prefs " + userPrefs);
 
         versionedModulePlanner = new VersionedModulePlanner(modulePlanner);
+
+        availableModules = new FilteredList<>(versionedModulePlanner.getAvailableModuleList());
```
[the remark this answers is not in the document]
Comment (Hilda-Ang) [reply src/main/java/seedu/planner/model/ModelManager.java]: Yeah, cause we're planning to implement filter

File: src/main/java/seedu/planner/model/ModulePlanner.java [hunk 2]
```diff
@@ -58,6 +72,14 @@ public ModulePlanner(ReadOnlyModulePlanner toBeCopied) {
         resetData(toBeCopied);
     }
 
+    /**
+     * Replaces the contents of the module list with {@code modules}.
+     * {@code modules} must not contain duplicate modules.
+     */
+    public void setAvailableModules(List<Module> modules) {
+        this.availableModules.setModules(modules);
```
[the remark this answers is not in the document]
Comment (Hilda-Ang) [reply src/main/java/seedu/planner/model/ModulePlanner.java]: Done

**Comment by Hilda-Ang** [comment]
Just to implement the available modules list in ModulePlanner, cause I was following the example in AddressBook. But if you feel don't need, I can remove it also

**Comment by GabrielYik** [comment]
I think we can leave it for now. Tomorrow or something can discuss

**Comment by Hilda-Ang** [comment]
Okay

**Pull request merged**